source/includes/_tenant.md

```diff
@@ -90,6 +90,19 @@ body = Tenant(api_key='demo_api_key', api_secret='demo_api_secret')
 tenantApi.create_tenant(body, created_by='demo')
 ```
 
+```javascript
+const api: killbill.TenantApi = new killbill.TenantApi(config);
+const tenant: killbill.Tenant = {apiKey: `api_key`, apiSecret: `api_secret`};
+const response: AxiosResponse<killbill.Tenant, any> = await api.createTenant(tenant, 'created-by');
+```
+
+```php
+$tenant = new Tenant();
+$tenant->setApiKey('php-tenant1');
```

Comment:
Maybe we should be consistent: either always specify something like `api_key`/`api_secret` or the default (`bob`/`lazar`)?

Reply:
I will fix this in the next PR, merging the PR to check how the docs render.